pkg/provisioner/dependency.go

```diff
@@ -38,6 +39,7 @@ var (
 	functions = map[string]ProvisionFunc{
 		kubeadmInstaller:          kubeadm,
 		kindInstaller:             kind,
+		NVKindInstaller:           nvkind,
```

Comment:
This should probably be:
```suggestion
		nvkindInstaller:           nvkind,
```